Curve-finance-suite/src/agents/remove.imbalance.liquidity.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from 'forta-agent';
+
+import abi from '../utils/stable.swap.abi';
+
+// @ts-ignore
+import abiDecoder from 'abi-decoder';
+abiDecoder.addABI(abi);
+
+export const REMOVE_LIQUIDITY_IMBALANCE_SIGNATURE =
+  'RemoveLiquidiityImbalance(address, uint256[3], uint256[3],uint256, uint256)';
+
+const createFinding = (alertID: string, address: string): Finding => {
+  return Finding.fromObject({
+    name: 'RemoveLiquidityImbalance funciton called',
+    description: 'RemoveLiquidityImbalance funciton called on pool',
```

Comment:
on what pool? Is the address field in metadata the pool? Can you add the pool in the metadata? Can you add any metadata to make this finding more informative?